ci/install_libseccomp.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# Copyright 2021 Sony Group Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+
+arch=$(uname -m)
+
+libseccomp_version="2.5.1"
+libseccomp_tarball="libseccomp-${libseccomp_version}.tar.gz"
+libseccomp_url="https://github.com/seccomp/libseccomp/releases/download/v${libseccomp_version}/${libseccomp_tarball}"
+libseccomp_install_dir="$1"
+cppflags="-O2"
```

Comment:
Unused? Should it be referenced in the hard-coded `-O2` below?

Reply:
Sorry, `cflags` is correct. I fixed it.